packet/utils.py

```diff
@@ -85,3 +85,10 @@ def wrapped_function(*args, **kwargs):
         return func(*args, **kwargs)
 
     return wrapped_function
+
+def notify_slack(name: str):
+    """
+    Sends a congratulate on sight decree to Slack.
+    """
+    msg = f'{name} got :100: on packet. Shower on sight.'
+    requests.put(app.config["SLACK_WEBHOOK_URL"], json={'text':msg})
```

Comment:
You may want to add a check here so it doesn't 500 if the config value is `None`. In prod it should always be configured but it would be nice to just log something for use when doing local debugging.